src/birthder.js

```diff
@@ -235,6 +237,62 @@
       msg.send(`*${title} list*\n${message.join('\n')}`)
     })
 
+    robot.respond(routes.agreeToPitchIn, async msg => {
+      const targetUsername = msg.match[1]
+      const bdayUser = await routines.findUserByName(robot, targetUsername)
+
+      if (!bdayUser) {
+        msg.send(`I have never met ${targetUsername}.`)
+
+        return
+      }
+
+      if (!bdayUser.birthdayChannel) {
+        msg.send('This person does not plan to celebrate his birthday in the near future. Do not rush.')
```

Comment:
This person does not plan to celebrate their birthday in the nears future. Do not rush.